clients/java/pom.xml

```diff
@@ -273,6 +273,41 @@
             <goals>
               <goal>generate</goal>
             </goals>
+            <configuration>
+              <!-- https://github.com/OpenAPITools/openapi-generator/tree/master/modules/openapi-generator-maven-plugin -->
+              <!-- https://github.com/OpenAPITools/openapi-generator/blob/master/docs/generators/java.md -->
+              <generatorName>java</generatorName>
+              <inputSpec>${openapi.dir}/rest-api.yaml</inputSpec>
+              <modelPackage>io.camunda.client.protocol.rest</modelPackage>
+              <typeMappings>OffsetDateTime=String</typeMappings>
+              <generateApis>false</generateApis>
+              <generateApiTests>false</generateApiTests>
+              <generateSupportingFiles>false</generateSupportingFiles>
+              <generateModels>true</generateModels>
+              <generateModelDocumentation>false</generateModelDocumentation>
+              <generateModelTests>false</generateModelTests>
+              <!-- validate the spec on every generation -->
+              <skipValidateSpec>false</skipValidateSpec>
+              <configOptions>
+                <additionalModelTypeAnnotations>@com.fasterxml.jackson.annotation.JsonInclude(com.fasterxml.jackson.annotation.JsonInclude.Include.NON_NULL)</additionalModelTypeAnnotations>
+                <documentationProvider>none</documentationProvider>
+                <enumUnknownDefaultCase>true</enumUnknownDefaultCase>
+                <library>apache-httpclient</library>
+                <java8>true</java8>
+                <openApiNullable>false</openApiNullable>
+                <serializationLibrary>jackson</serializationLibrary>
+                <useReflectionEqualsHashCode>false</useReflectionEqualsHashCode>
+              </configOptions>
```

Comment:
🔧 You should be able to move the configuration from the `execution` level to the `plugin` level. You can then just overwrite the `modelPackage` in he configuration section of the "old-rest" execution. That removes duplication.

Reply:
Let's also use V1 REST YAML to generate the "old-rest" models.